### ISSUE TITLE
Whitelisting in premium 

### ISSUE THREAD

**Issue by Am1892m** [tomkahn/lockmyphone_public#12]
Hi. 
The premium version does not have white listing of apps. I wanted to recommend this feature.
Thanks.

**Comment by ahivarn** [comment]
I would argue that it would dilute the purpose of the app. What instead i would suggest is to allow calling feature while locked. Maybe to select emergency contacts.

**Comment by tomkahn** [comment]
Calling emergency contacts is already possible. Please see [this video](https://www.youtube.com/watch?v=dE_bbD5vXDU).

You have to have some kind of lock screen enabled for it to work, though. Whitelisting individual apps would be a great feature to have, though.

**Comment by nuraysalim** [comment]
Personally, I think that whitelisting would bring the app away from its purpose. The purpose is to lock your phone so you do not use it. It is not a productivity app, to restrict access from certain apps and websites which there are plenty of. I think the fact that after you lock your phone - it is lock and that's it is the feature that makes this app unique and irreplacable.

**Comment by cristalcommons** [comment]
I agree with nuraysalim. Still, maybe we can consider some kind of integration with blocking apps (e.g. AppBlock, StayFocused), time and task management apps (e.g. TickTick, TimeTune), or habit tracking apps could be done.

E.g. Making apps share a common calendar called "Strict Focus", if the user has created on Todoist app the task of studying next 3 days from 15:00 to 17:00 and adds it to this Strict Focus calendar, it will be automatically synced with Lock Your Phone and lock the phone during the time the calendar event "Study for next exam" is active, without having to create a block for it and delete it 3 days after.
